plugin/plugin.go

```diff
@@ -141,13 +168,47 @@ func Exec(ctx context.Context, args Args) error {
 				URL:                  deeplink,
 				LastUpdated:          time.Now(),
 				PipelineID:           args.Name,
-				IssueKeys:            []string{issue},
+				IssueKeys:            issues,
 				State:                state,
 				UpdateSequenceNumber: args.Build.Number,
+				References: []struct {
```

Comment:
add a check to make sure the branch and URI values exist before adding them to the References so it'll only be added when the necessary values are available